web/src/components/EmojiReaction/EmojiReaction.js

```diff
@@ -107,17 +108,20 @@ const EmojiReaction = ({
           horizontal: 'left',
         }}
       >
-        <div className="p-2 pr-3 flex">
-          {emojiMenu.map((emoji, i) => (
-            <button
-              className="p-2"
-              style={textShadow}
-              key={`${emoji}-${i}}`}
-              onClick={() => handleEmojiClick(emoji)}
-            >
-              {emoji}
-            </button>
-          ))}
+        <div className="p-2 pr-3 flex flex-col">
+          <div className="inline-flex">
+            {emojiMenu.map((emoji, i) => (
+              <button
+                className="p-2"
+                style={textShadow}
+                key={`${emoji}-${i}}`}
+                onClick={() => handleEmojiClick(emoji)}
+              >
+                {emoji}
+              </button>
+            ))}
+          </div>
+          <button onClick={onShowPartReactions}>View Reactions</button>
```

Comment:
I've rarely just straight black text in the app, so lets add `className="text-gray-700"` for consistency.